# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/locales/en-US/chrome/dom/dom.properties
# Commit: 7ea2efe7df94
# Full Hash: 7ea2efe7df94b7cfb09b2033fc4e28e2d6d4efa8
# Author: Ziran Sun <zsun@igalia.com>
# Date: 2023-05-20 09:21:40
# Regressor Bug: 1832837
# File Overlap Count: 1
# Description:
#   Bug 1832837 - Remove the fullscreen element ready check on fullscreen element descendant. r=edgar
#   
#   https://fullscreen.spec.whatwg.org/#fullscreen-element-ready-check
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D177913
# ==============================================================================

diff -r 846df2d0be58 -r 7ea2efe7df94 dom/locales/en-US/chrome/dom/dom.properties
--- a/dom/locales/en-US/chrome/dom/dom.properties	Fri May 19 06:59:48 2023 +0000
+++ b/dom/locales/en-US/chrome/dom/dom.properties	Fri May 19 10:14:37 2023 +0000
@@ -72,7 +72,6 @@
 FullscreenDeniedLostWindow=Request for fullscreen was denied because we no longer have a window.
 FullscreenDeniedPopoverOpen=Request for fullscreen was denied because the element is already open as a popover.
 FullscreenDeniedSubDocFullscreen=Request for fullscreen was denied because a subdocument of the document requesting fullscreen is already fullscreen.
-FullscreenDeniedNotDescendant=Request for fullscreen was denied because requesting element is not a descendant of the current fullscreen element.
 FullscreenDeniedNotFocusedTab=Request for fullscreen was denied because requesting element is not in the currently focused tab.
 FullscreenDeniedFeaturePolicy=Request for fullscreen was denied because of FeaturePolicy directives.
 FullscreenExitWindowFocus=Exited fullscreen because a window was focused.